common/app/experiments/Experiments.scala

```diff
@@ -23,6 +23,15 @@ object LiveblogRendering
       participationGroup = Perc50,
     )
 
+object FrontRendering
+    extends Experiment(
+      name = "front-rendering",
+      description = "Use DCR for fronts",
+      owners = Seq(Owner.withGithub("dotcom")),
```

Comment:
Not sure if this is actually allowed?

Reply:
You can always just make yourself the owner!

Reply:
(but I don't know if it actually checks if it's a real username)

Reply:
I have seen it written like that: https://github.com/guardian/frontend/blob/791fe19364cd2b3e52a34632fe3b09e3eaf19d93/common/app/conf/switches/FeatureSwitches.scala#L84